lang/fpc/files/fpc.cfg.patch

```diff
@@ -1,5 +1,17 @@
 --- etc/fpc.cfg	2019-06-14 15:54:16.000000000 +0200
 +++ fpc.cfg-new	2019-08-17 10:54:57.000000000 +0200
+@@ -175,6 +175,11 @@
+ # searchpath for tools
+ -FD/opt/local/libexec/fpc/lib/fpc/3.2.2/bin/$FPCTARGET
+ 
++# searchpath for qt4pas framework
++#IFDEF Darwin
++-FfPREFIX/libexec/qt4/lib
++#ENDIF
++
+ # path to the gcclib
+ #ifdef cpui386
+ -Fl/Library/Developer/CommandLineTools/usr/lib/clang/14.0.3/lib/darwin
```

Comment:
Admittedly, I may be unaware of the context here, but maybe set Clang version via a macro (with reinplace)? So that it can be set in the portfile in an obvious way.

Reply:
I have to review this. On first glance, this might well be an accidental addition. The linker issue with macOS 13 and Xcode 15 has manifested on the builders and that needs more work anyway. Although not related, there will be so much more work required, that I will change this PR as work in progress and keep individual commits instead of squashing them. I think, I have just found a way, how to patch the sources to avoid the massive alignment problem when linking.

Reply:
This is part of the base version of the config file as created by the default creation script for the config file. So far, there was no reason to change it and therefore, I would leave it as it is.

Reply:
> Just to make sure: Are you aware, that this line is not changed in the patch, but just serves to locate the lines?

@kamischi Yes, of course, but whatever in the source has `-Fl/Library/Developer/CommandLineTools/usr/lib/clang/14.0.3/lib/darwin` for all i386 looks incorrect. I am not saying the patch is wrong. Apparently the source is wrong and should be patched.